secretstores/aws/secretmanager/secretmanager.go

```diff
@@ -109,11 +109,11 @@ func (s *smSecretStore) BulkGetSecret(req secretstores.BulkGetSecretRequest) (se
 				// VersionStage: versionStage,
 			})
 			if err != nil {
-				return secretstores.GetSecretResponse{Data: nil}, fmt.Errorf("couldn't get secret: %s", *entry.Name)
+				return secretstores.BulkGetSecretResponse{Data: nil}, fmt.Errorf("couldn't get secret: %s", *entry.Name)
```

Comment:
Not changed in this PR but what's the rationale for `VersionId` and `VersionStage` to be commented out on L107 & L108? This seems like the kind of thing that should really be documented with a comment rather than "spooky ghost code"

Reply:
👻

Reply:
I removed the comments, indeed it was👻 code!